data/field.go

```diff
@@ -196,6 +196,13 @@ func (f *Field) Set(idx int, val interface{}) {
 	f.vector.Set(idx, val)
 }
 
+// SetConcreateAt sets the Fields's value to index idx to the val
+// if the Field is not nullable or to the pointer to val if it is nullable
```

Comment:
```suggestion
// SetConcreteAt sets the Fields's value at index idx to val for of val.
// val must be a non-pointer type or a panic will occur.
// If the underlying FieldType is nullable it will set val as a pointer to val. If the FieldType
// is not nullable, then this method behaves the same as the Set method.
```